data/distro.yml

```diff
@@ -405,6 +405,7 @@
         <pre><code>
           <span class="unselectable">#</span> flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
         </code></pre>
+        <p><b>Important note:</b> As of March 2021, Raspberry Pi computer still ship with the 32-bit version of Raspberry Pi OS. However Flathub started phasing out support for that architecture. If you consider Flathub as an important source of applications, it is recommended to use Raspberry Pi OS 64-bit as newer application are more likely to available, and other one updated.</p>
```

Comment:
"as newer application are more likely to available, and other one updated." - This does not sound right to me.

Reply:
yeah missing words. Will fix. 

(and I re-read twice before submitting)